ktxspec.adoc

```diff
@@ -1148,11 +1147,14 @@ have a sample with a channelType that indicates it is alpha.
 ===== DFD for Supercompressed Data
 When `<<_supercompressionscheme,supercompressionScheme>>` is not 0
 the _dfDescriptorBlock_ must preserve the `colorModel`, `transferFunction`,
-`colorPrimaries`, `flags` and `texelBlockDimension[0-3]` of the
-pre-deflation images along with each sample's `channelType`,
-`qualifiers`, `bitlength`, `bitOffset`, `sampleLower` and `sampleUpper`.
-`bytesPlane[0-7]` must be set to 0 to indicate an _unsized format_,
-as described in Section 5.19 _Unsized Formats_ of <<KDF13>>.
+`colorPrimaries`, `flags`, `texelBlockDimension[0-3]` and
+`bytesPlane[0-7]` of the pre-deflation images along with each
+sample's `channelType`, `qualifiers`, `bitlength`, `bitOffset`,
+`sampleLower`, and `sampleUpper`.
+
+BasisLZ supercompression works only on ETC1S blocks so
+`texelBlockDimension[01]` must be 3 and `bytesPlane0` must be
+1 or, if there is an alpha plane, 2.
```

Comment:
Pre-deflated length of ETC1S blocks is either 8 or 16.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Specification/pull/221/commits/c5c9f2efa618d7d4135e9e98529ad77b1e230abd.